ott/geometry/costs.py

```diff
@@ -94,6 +94,23 @@ def pairwise(self, x, y):
     return -2 * dot(x, y)
 
 
+@jax.tree_util.register_pytree_node_class
+class Cosine(CostFn):
+  """Cosine distance distance CostFn."""
+
+  def __init__(self, eps=1e-8):
+    super().__init__()
+    self._eps = eps
```

Comment:
eps (or epsilon) is used elsewhere in the code for regularization. Maybe give this a more explicit name, such as ridge, or avoid_zero etc..

Reply:
fixed with a new, explicit name 'ridge'